jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/SessionHandler.java

```diff
@@ -613,12 +614,19 @@ public void doHandle(String target, Request baseRequest, HttpServletRequest requ
      * CookieConfig
      *
      * Implementation of the jakarta.servlet.SessionCookieConfig.
-     * SameSite configuration can be achieved by using setComment
+     * SameSite configuration can be achieved by using setComment.
+     * Partitioned configuration can be achieved by using setComment.
      *
      * @see HttpCookie
      */
     public final class CookieConfig implements SessionCookieConfig
     {
+        protected static final String SAME_SITE_COMMENT = "__SAME_SITE_";
```

Comment:
These statics are already part of `org.eclipse.jetty.ee9.nested.Response.HttpCookieFacade`

We should use the static methods on that class instead of copying them here.

These should just be made public and used in the SessionHandler.

* `private static SameSite getSameSiteFromComment(String comment)`
* `private static boolean isPartitionedInComment(String comment)`
* `private static boolean isHttpOnlyInComment(String comment)`

Reply:
Done.